cfe_internal/update/update_policy.cf

```diff
@@ -26,6 +26,9 @@ bundle agent cfe_internal_update_policy
       "have_user_specified_update_bundle"
         expression => isvariable( "def.mpf_update_policy_bundle" );
 
+      # Define class if def.policy_servers is present for selecting the first available policy server
+      "have_def_policy_servers"
```

Comment:
Won't we have this variable all the time? it will either be one or more, so I think the coniton sould be based on length of the list.

Reply:
todo: yank all this failover stuff, only include defaulting `sys.policy_servers`

Reply:
right, yank this. no need.